.cirrus.yml

```diff
@@ -119,6 +119,13 @@ preflight_check_task:
 
 # Pipeline 2
 
+ubuntu_22_04_x64_min_version_debug_task:
```

Comment:
Since we don't have unlimited CI time on Cirrus CI I'm wondering whether we should limit MSRV test to only one target?

Reply:
When I manually checked the min version, I realized that different platforms have different minimal rust version due to conditional dependencies.

So my idea was that we have a min rust version check for every OS/arch pair.

I think the risk is low that the min rust version differs on any unix derivate so it should be reasonable safe to test it once for ubuntu or archlinux and skip freebsd and mac os.

But windows has a lot of additional dependencies so I would at least run this additionally.

So my minimum recommendation would be:
* one additional min target on linux
* one min target on windows

Would you be fine with it?

Reply:
Hahaha, that was also what I thought after submitting the review. Yes, having one for POSIX like systems and one for Windows should be good. If we encounter issues with the MSRV we can add more CI later on.

Another idea would be to conditionally add more platforms when `Cargo.toml` changes and have only the two for regular PRs